packages/@icon-magic/distribute/src/create-sprite.ts

```diff
@@ -55,7 +74,10 @@ async function appendIcon(parent: Element, asset: Asset): Promise<void> {
  */
 export async function appendToSvgDoc(asset: Asset, doc: Document, svgEl: SVGSVGElement, category: string): Promise<void> {
   if (category) {
-    let def = doc.getElementById(category);
+    // TODO: Replace this with findElementById, which right now doesn't find the <defs> with the ID
```

Comment:
Is it not supported in xmldom?

Reply:
It is, and the method calls, it just returns `null` every time, not sure why.